src/robotlibcore.py

```diff
@@ -142,6 +135,37 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, keyword_name):
+        method = self.keywords.get(keyword_name)
+        if not method:
+            raise ValueError('%s is not keyword.' % keyword_name)
+        robot_types = self._get_robot_types(method)
+        if robot_types is not False:
+            if robot_types is None:
+                return {}
+            return robot_types
+        return self._get_annotations(method)
+
+    def _get_robot_types(self, method):
+        if hasattr(method, 'robot_types'):
+            robot_types = method.robot_types
+            if robot_types == tuple():
+                return False
+            return robot_types
```

Comment:
As commented above, and assuming I understood this correctly, this whole method can be removed.